src/platform/site-wide/user-nav/tests/user-nav.cypress.spec.js

```diff
@@ -0,0 +1,26 @@
+import Timeouts from 'platform/testing/e2e/timeouts';
+
+const selectors = {
+  menu: '#login-root button[aria-controls="account-menu"]',
+  signOut: '#account-menu ul li:nth-child(4) a',
+};
+
+describe('User Nav Test', () => {
+  it('Displays the proper elements on the nav', () => {
+    cy.login();
+    cy.visit('/my-va');
+    cy.title().should('contain', 'My VA | Veterans Affairs');
+    cy.get(selectors.menu, { timeout: Timeouts.slow })
+      .should('be.visible')
+      .then(dropDownMenu => {
+        cy.wrap(dropDownMenu).click();
+        cy.get(selectors.signOut, { timeout: Timeouts.slow })
+          .should('be.visible')
+          .and('contain', 'Sign Out');
+        // .then(signOutButton => {
```

Comment:
If this is commented out, should it be removed?

Reply:
There seems to be a problem with cross-origin testing in Cypress that did not exist in Nightwatch.  I'm currently attempting to come up with a different way to test this button accurately, however, due to the volume of this branch I didn't want to hold up the other tests moving forward in the interim (this will be solved before it's Nightwatch counterpart is removed).

Reply:
Gotcha 👍